examples/b2c/checkout/integrations/classes/B2CDeliverySampleTest.cls

```diff
@@ -11,16 +21,21 @@ private class B2CDeliverySampleTest {
         WebCart cart = new WebCart(Name='Cart', WebStoreId=webStore.Id, AccountId=account.Id);
         insert cart;
 
-        CartDeliveryGroup cartDeliveryGroup = new CartDeliveryGroup(CartId=cart.Id, Name='Default Delivery 1');
-        insert cartDeliveryGroup;
+        for (Integer i = 1; i <= cartDeliveryGroupsNo; i++) {
+            CartDeliveryGroup cartDeliveryGroup = new CartDeliveryGroup(CartId=cart.Id, Name='Default Delivery ' + i);
+            insert cartDeliveryGroup;
 
-        CartItem cartItem = new CartItem(CartId=cart.Id, Type='Product', Name='TestProduct', CartDeliveryGroupId=cartDeliveryGroup.Id);
-        insert cartItem;
+            for (Integer j = 0; j < expectedCDGMInTheIntegrationMock; j++) {
+                CartItem cartItem = new CartItem(CartId=cart.Id, Type='Product', Name='TestProduct', CartDeliveryGroupId=cartDeliveryGroup.Id);
+                insert cartItem;
+            }
+        }
     }
 
 
     @isTest static void testIntegrationRunsSuccessfully() {
         Test.startTest();
+        init();
```

Comment:
init() is also called from setup()

Reply:
@gurpreetsainisalesforce I'm aware, unfortunately those comot 'static' variables in apex are actually 'null' if not explicitly populated; may be it use a different instance to run the setup and to run the test - not sure, anyway it is null whence the init is not called from the test